### PR TITLE
CXP-962: Forbid Repositories in backend, only use Queries > Connected Apps

### PR DESCRIPTION
**Pull request by masacc**
**Description (for Contributor and Core Developer)**

In Connected Apps scope, remove repositories and replace them by queries

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: src/Akeneo/Connectivity/Connection/back/Domain/Apps/Persistence/Query/FindAllConnectedAppsQueryInterface.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Domain\Apps\Persistence\Query;
+
+use Akeneo\Connectivity\Connection\Domain\Apps\Model\ConnectedApp;
+
+interface FindAllConnectedAppsQueryInterface
```
Comment (RiffVentura) [inline src/Akeneo/Connectivity/Connection/back/Domain/Apps/Persistence/Query/FindAllConnectedAppsQueryInterface.php:8]:
```suggestion
interface GetConnectedAppsQueryInterface
```
Existing interfaces in the same namespace are using `get`, `delete` `save` `create`
same comment for find* queries

File: src/Akeneo/Connectivity/Connection/back/Domain/Apps/Persistence/Query/CreateConnectedAppQueryInterface.php [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?php
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Domain\Apps\Persistence\Query;
```
Comment (RiffVentura) [inline src/Akeneo/Connectivity/Connection/back/Domain/Apps/Persistence/Query/CreateConnectedAppQueryInterface.php:4]:
```suggestion
namespace Akeneo\Connectivity\Connection\Domain\Apps\Persistence;
```
I'm wondering if it's an opportunity to update namespaces for our queries and remove needless indirections

**Review by RiffVentura — APPROVED** [review]

**Review by guillaumefer-akeneo — APPROVED** [review]

File: src/Akeneo/Connectivity/Connection/back/Domain/Apps/Persistence/Query/FindAllConnectedAppsQueryInterface.php (hunk 1 above)
Comment (willy-ahva) [reply src/Akeneo/Connectivity/Connection/back/Domain/Apps/Persistence/Query/FindAllConnectedAppsQueryInterface.php]: Seen IRL, we'll see later

**Review by tseho — APPROVED** [review]

**Comment by masacc** [comment]
warmup

**Review by jmleroux — APPROVED** [review]
ok for migration

**Pull request merged**